scico/loss.py

```diff
@@ -260,8 +271,7 @@ class PoissonLoss(Loss):
     Poisson negative log likelihood loss
 
     .. math::
-        \mathrm{scale} \cdot \sum_i [A(x)]_i - y_i \log\left( [A(x)]_i \right) + \log(y_i!)
-
+        \mathrm{scale} \sum_i [A(x)]_i - y_i \log\left( [A(x)]_i \right) + \log(y_i!)
```

Comment:
Hate to say it: this looks odd to my eye. I think the cdot was intended to fix the oddness of `scale` being a word while the rest are math symbols. Elsewhere we have used `scale` for the name in code and \alpha in the math, see, e.g., https://scico--78.org.readthedocs.build/en/78/_autosummary/scico.functional.html#scico.functional.ScaledFunctional

Reply:
I went with `\alpha`+`scaling parameter` in the docs, `scale` in the code now. (86e12ca)
@bwohlberg @Michael-T-McCann let me know what you think.

Reply:
That looks much better.